portfolio/src/main/webapp/template.html

```diff
@@ -68,73 +68,39 @@ <h2 class="heading">Latest Projects</h2>
                             <div class="item featured text-center">
                                 
                                 <div class="featured-image has-ribbon">
-                                    <a href="https://themes.3rdwavemedia.com/bootstrap-templates/startup/launch-bootstrap-4-template-for-saas-businesses/" target="_blank">
-                                    <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/project-featured.jpg" alt="project name" />
+                                    <a href="#" target="_blank">
+                                    <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/GradTracks.png" alt="GradTracks" />
```

Comment:
Per the style guide, break long lines to improve readability here and everywhere:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping